tests/telemetry/test_telemetry.py

```diff
@@ -12,7 +12,16 @@
 logger = logging.getLogger(__name__)
 
 TELEMETRY_PORT = 50051
+mode_subscribe = "subscribe"
+mode_get = "get"
 
+subscribe_mode_stream = 0
+subscribe_mode_once = 1
+subscribe_mode_poll = 2
+
+submode_targetdefined=0
+submode_onchange=1
+submode_sample=2
```

Comment:
Can you use all upper case for global constants?

Reply:
Ack